README.md

```diff
@@ -18,11 +22,9 @@ to send a pull request.
 
 ### Adding a translation
```

Comment:
Let's add a section talking about how to run our site in development mode. Do you run `docker-compose up -d`? How do I see my changes?